### PR TITLE
show individual level csr as integer

### PR DESCRIPTION
**Pull request by RyanEddyIC**
IVL-178727386

### CODE REVIEW

File: app/controllers/hbx_admin_controller.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ def registry
   end
 
   def edit_aptc_csr
-    raise NotAuthorizedError if !current_user.has_hbx_staff_role?
+    # raise NotAuthorizedError if !current_user.has_hbx_staff_role?
```
Comment (ymhari) [inline app/controllers/hbx_admin_controller.rb:12]: Why are we removing this?

**Review by ymhari — CHANGES_REQUESTED** [review]

**Review by jayreddy519 — DISMISSED** [review]
changes to display individual level csr.

LGTM.

make sure GHA is green before merge.

**Comment by markgoho** [comment]
@RyanEddyIC please re-open if this still relevant

File: features/step_definitions/edit_aptc_csr_steps.rb [hunk 2]
```diff
@@ -26,6 +26,15 @@
   find_link('Edit APTC / CSR').visible?
 end
 
+Then(/Hbx Admin clicks the Edit APTC CSR link/) do
+  click_link 'Edit APTC / CSR'
```
Comment (markgoho) [inline features/step_definitions/edit_aptc_csr_steps.rb:30]: remove hard-coded text

File: features/step_definitions/edit_aptc_csr_steps.rb [hunk 3]
```diff
@@ -26,6 +26,15 @@
...
+end
+
+Then(/Hbx Admin should see individual level csr percent/) do
+  expect(page).to have_content("CSR % as Integer")
```
Comment (markgoho) [inline features/step_definitions/edit_aptc_csr_steps.rb:34]: hard-coded english text

**Review by markgoho — CHANGES_REQUESTED** [review]
remove hard-coded english text from tests

File: features/support/object_model_pages/admin/edit_aptc.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+#admin/families tab/ edit aptc action
+class EditAptc
+
+  def self.csr_pct_as_integer
+    '[data-aptc-cucumber="csr_pct"]'
```
Comment (markgoho) [inline features/support/object_model_pages/admin/edit_aptc.rb:7]: let's use `data-cuke` or `data-cucumber` for consistency

**Review by markgoho — DISMISSED** [review]
the `data-` attribute we use should be consistent, but I think it's only you and @jayreddy519 that have used them

`data-cuke` is shorter, maybe let's go with that?

File: features/support/object_model_pages/admin/edit_aptc.rb (hunk 4 above)
Comment (RyanEddyIC) [reply features/support/object_model_pages/admin/edit_aptc.rb:7]: Alright, didn't realize we were going for a universal testing attribute but that makes sense.

File: app/controllers/hbx_admin_controller.rb (hunk 1 above)
Comment (markgoho) [reply app/controllers/hbx_admin_controller.rb]: @RyanEddyIC I see you resolved this conversation - can we make sure questions are answered here on GitHub?

**Review by ymhari — APPROVED** [review]

**Review by jayreddy519 — COMMENTED** [review]
.

**Pull request merged**